### DISCUSSION TITLE
Able to receive data but unable to send data to Easytouch via RS485

### DISCUSSION THREAD

**Discussion by FlaMike** [tagyoureit/nodejs-poolController#588] [RS485] [answered]
I just installed the nodejs-poolController on a pi 3b+.  The pi is in my house and connected to the Easytouch is via Cat6 with RJ45 to terminal screw adapters at the controller end and adjacent to the pi.  A Moyina USB to RS485 converter is in place at the pi. 

I am following the guide posted at Trouble Free Pools for the installation.  When I start the nodejs-poolController in step E 3 of the guide all seems to be going well.  However, the RS485 stats show the pi is able to receive data but cannot send any data to the Easytouch.  The console shows "collsions" may be the problem.  Because of the communication failures,  the configuration of the nodejs-poolController never completes.

I have double-checked the wiring and it appears to be correct and OK.  (The yellow wire from the terminal block in the Easytouch controller goes to A+ on the converter and the green wire from the Easytouch terminal block goes to B- on the converter.)

I tried switching the yellow & green wires, just in case, to no avail.  Adding the ground wire from the Easytouch to the converter prevents any communication from occurring at all.

I've also added a 2 foot USB extension cord for the USB to RS 485 converter in case there is electrical noise from the pi causing interference.  Still no love.

Does anyone have suggestions on how to overcome this roadblock?

Thanks in advance for your help!!

**Comment by tagyoureit** [comment, marked as answer]
Check out the [RS-485 wiki](https://github.com/tagyoureit/nodejs-poolController/wiki/RS-485-Adapter-Details).  If that doesn't work you can attach a [replay](https://github.com/tagyoureit/nodejs-poolController/wiki/How-to-capture-all-packets-for-issue-resolution) (and screenshots or pictures) to help us assess the situation.

**Comment by FlaMike** [comment]
Thank you for your speedy response!!  I went through the wiki again.  I also checked every wire & upgraded some from Cat6 to slightly heavier gauge solid core wire.  Finally, I reinstalled everything from scratch, including the OS on the pi.  While I am not sure what seems to have solved the problem (given the number of changes I made) but the bottom line is the pool controller is now working!  The pool controller software recognized the Easytouch controller and I was able to start the dash panel software.

While I am still working through some issues,  I will check the existing posts before I open another discussion.  (In short though, the web interface has placeholders for items and the correct background image, but all fields within the panel are blank.)  It's rough being a newbie!

Thanks again for your help and very interesting software!!

**Reply by tagyoureit** [reply]
If you go to the gears (setup, upper right) do the right values appear where they are missing in the home page?

**Comment by FlaMike** [comment]
There are no gears on the screen, see attached.

Do I need to change the IP address in the dashpanel config.json. I am 
not viewing the dashpanel from the pi?
On 9/4/22 4:22 PM, tagyoureit wrote:
 >
 > If you go to the gears (setup, upper right) do the right values 
appear where they are missing in the home page?
 >
 > —
 > Reply to this email directly, view it on GitHub, or unsubscribe.
 > You are receiving this because you authored the thread.Message ID: 
***@***.***>

PoolControllerDashboard.png
